data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #74.    #@markdown KEEP_GROUPING: If True, the rows will remain unique GROUPING_COLUMNS values. Otherwise
</pre><p> </p><p>This is probably very clear to you since you've been staring at this for awhile, but per our chat this was not as clear to me. Would you be willing to consider something like "If KEEP_GROUPING is True, the exported rows will have unique values of the grouping columns selected above. If False, the exported rows will still contain the mean, variance, and other statistics for each group, but the original set of rows will be exported without combining any rows.", which came from the last round of our iterative refinement on Chat?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>